src/controls/FormBuilder.ts

```diff
@@ -120,9 +120,10 @@ export class FormBuilder {
             // from SharedWidgetProps
             label?: string | false
             requirements?: Requirements[]
+            collapsible?: boolean
```

Comment:
but also, I think I'd prefer to fix that once forall rather than make widget have collapsible configurable; hm.. I'll take a look asap